ekundayo-ab/hello-books
# #152954694 Integrate client side tests

### What does this PR do?
Ensure client side tests is integrated for actions, reducers and components (not all yet, still on!)

### Description of Task to be completed?
Write tests for all actions, reducers and components (still on!)

### How should this be manually tested?
1. Pull in latest changes from remote repository to local repository
2. Fire up terminal run `npm install`
3. run `npm run test-client` to see testing in action
4. Observe and study tests report

### Any background context you want to provide?
Chore is still on!

### What are the relevant pivotal tracker stories?
#152954694 Integrate client side tests

### Screenshots (if appropriate)
N/A

### Questions:
N/A
